init.lua

```diff
@@ -189,9 +189,8 @@ minetest.register_entity("painting:paintent", {
 
 		--get player eye level
 		--see player.h line 129
-		local ppos = puncher:getpos()
-
-		ppos.y = ppos.y + 1.625
+		local ppos = vector.add(puncher:getpos(), puncher:get_eye_offset())
```

Comment:
should be `get_pos()` since 0.4.16.

Reply:
I'm surprised I didn't catch that. Will get it updated.

Reply:
Done in dc43218